CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ BUG FIXES:
 * core: Fix a bug that prevents certain loggers from writing to a log file. [[GH-29917](https://github.com/hashicorp/vault/pull/29917)]
 * identity: reintroduce RPC functionality for group creates, allowing performance standbys to handle external group changes during login and token renewal [[GH-30069](https://github.com/hashicorp/vault/pull/30069)]
 * plugins (enterprise): Fix plugin registration with artifact when a binary for the same plugin is already present in the plugin directory.
+* raft/retry_join: Fix decoding `auto_join` configurations that include escape characters. [[GH-29874](https://github.com/hashicorp/vault/pull/29874)]
```

Comment:
```suggestion
* core/raft: Fix decoding `auto_join` configurations that include escape characters. [[GH-29874](https://github.com/hashicorp/vault/pull/29874)]
```

I think this should be core/raft, looking at other changelog entries. I don't think we specify down to the retry_join level, but I also don't think it's storage/raft. If you change it it should technically move up to line 38.